tmva/tmva/src/BDTEventWrapper.cxx

```diff
@@ -65,3 +65,13 @@ Double_t BDTEventWrapper::GetCumulativeWeight(Bool_t type) const {
    if(type) return fSigWeight;
    return fBkgWeight;
 }
+
+////////////////////////////////////////////////////////////////////////////////
+/// Index of the variable to sort on
+
+Int_t& BDTEventWrapper::GetVarIndex() {
+   // This is a workaround for OSx where static thread_local data members are
```

Comment:
Anyway, for order of initialization reason.  A function static is usually preferable to a class static.